templates/application-kube-prometheus-stack.yaml

```diff
@@ -67,6 +69,14 @@ spec:
             podMonitorNamespaceSelector: {}
             podMonitorSelectorNilUsesHelmValues: false
             logLevel: debug
+            storageSpec:
+              volumeClaimTemplate:
+                spec:
+                  accessModes:
+                    - ReadWriteOnce
+                  resources:
+                    requests:
+                      storage: 50Gi
```

Comment:
@fernandoataoldotcom can you make this a value in the values.yaml? I assume we may want to change it from 50Gi to something else later. but for now could just add it in as an @ignored value